lib/suc.libjsonnet

```diff
@@ -25,9 +25,10 @@ local params = inv.parameters.system_upgrade_controller;
           force: true,
         },
         channel: channel,
+        version: version,
```

Comment:
According to the documentation:

> Providing a value for `version` will prevent polling/resolution of the `channel` if specified.

Does this still apply if `version=''` is supplied? If so, it probably makes more sense to do

```suggestion
        [if version != null then 'version']: version,
```

and set `version` to `null` when it's not provided.

Reply:
I' will check this

Reply:
setting `version` to `''` works and the channel is used but could also be left out if not set. I'll shorty test this suggegstion also with the component and not only by creating the plan by myself and the add the commit.